.circle/tests.sh

```diff
@@ -43,7 +43,7 @@ case ${CIRCLE_NODE_INDEX} in
     ;;
   1)
     fmriprep-docker -i poldracklab/fmriprep:latest --help
-    fmriprep-docker -i poldracklab/fmriprep:latest --config $HOME/nipype.cfg -w $HOME/ds005/scratch $HOME/data/ds005 $HOME/ds005/out participant --output-space fsaverage5 --debug --write-graph
+    fmriprep-docker -i poldracklab/fmriprep:latest --config $HOME/nipype.cfg -w $HOME/ds005/scratch $HOME/data/ds005 $HOME/ds005/out participant --template --debug --write-graph --use_aroma
```

Comment:
Sorry, I was unclear. `--template` is for specifying a *different* template. This will think it's getting `--template "--debug"`. Just drop the `--template` flag altogether.